backend/dal/Migrations/Initial/PostDeploy/01-ProjectStatus.sql

```diff
@@ -149,7 +149,7 @@ INSERT INTO dbo.[ProjectStatus] (
     13
```

Comment:
Depending on whether Jonny's PR merges first you may need to resolve conflicts here.

Reply:
I'll adjust them on mine! Still not 100% ready to merge mine yet